code/game/turfs/open/_open.dm

```diff
@@ -45,6 +45,18 @@
 	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
 	tiled_dirt = TRUE
 
+/turf/open/indestructible/floor
+	name = "floor"
+	icon = 'icons/turf/floors.dmi'
+	icon_state = "floor"
+	footstep = FOOTSTEP_FLOOR
+	barefootstep = FOOTSTEP_HARD_BAREFOOT
+	clawfootstep = FOOTSTEP_HARD_CLAW
+	heavyfootstep = FOOTSTEP_GENERIC_HEAVY
+	tiled_dirt = TRUE
+	explosion_block = 50
+
+
```

Comment:
Please put this in a seperate file in the nsv13 folder

Reply:
![image](https://user-images.githubusercontent.com/68692362/171921137-5122912d-a034-4a01-8fef-d0b816ea789c.png)